components/empty-state/empty-state-simple.js

```diff
@@ -35,32 +30,31 @@ class EmptyStateSimpleButton extends RtlMixin(LitElement) {
 		this._validatingDescriptionTimeout = null;
 	}
 
+	connectedCallback() {
+		super.connectedCallback();
+		this.addEventListener('d2l-empty-state-illustrated-check', this._handleEmptyStateIllustratedCheck);
```

Comment:
Could we just not wire up the event handler here? It'll bubble all the way to the top of the document and just get ignored, but as long as the illustrated code handles an `undefined` result (or maybe it could default it to `false` before firing the event), it should be fine.

Reply:
Yeah, I considered this as well but thought it better to not have the event bubble up to the rest of the document. If the event bubbling isn't a big deal, then it makes sense to simplify it.

Reply:
It's really low cost, and actually probably cheaper than handling it here. Mostly I'm thinking if we have a bunch of other empty-state types or actions in the future, we wouldn't want to filth them all up with this event handling code that really only makes sense in the illustrated + button case.